src/SFA.DAS.Tools.Servicebus.Support.Infrastructure/Services/ServiceBus/AsbService.cs

```diff
@@ -126,8 +126,20 @@ public async Task SendMessagesAsync(IEnumerable<QueueMessage> messages, string q
 
             if (messages.Count() > 0)
             {
-                var orginalMessages = messages.Select(m => m.OriginalMessage).ToList();
-                await messageSender.SendAsync(orginalMessages);
+                var orginalMessages = new List<Message>();
+                try
+                {
+                    orginalMessages = messages.Select(m => m.OriginalMessage).ToList();
+                    await messageSender.SendAsync(orginalMessages);
+                }catch(MessageSizeExceededException)
```

Comment:
We should capture the error and log it and format the line nicely
```suggestion
                }
                catch(MessageSizeExceededException)
```